be/src/common/status.h

```diff
@@ -76,6 +76,7 @@ namespace ErrorCode {
     TStatusError(HTTP_ERROR, true);                       \
     TStatusError(TABLET_MISSING, true);                   \
     TStatusError(NOT_MASTER, true);                       \
+    TStatusError(TRY_LOCK_FAILED, false);                 \
```

Comment:
add it somewhere to avoid stacktrace

Reply:
The second argument, false, indicates that the status does not print a stack trace.